documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -15,6 +15,10 @@ The state (enabled or disabled) of a feature gate may vary by default, depending
 As a feature gate graduates and reaches General Availability (GA), it transitions to an enabled state by default and becomes a permanent part of the Strimzi deployment.
 A feature gate at the GA stage cannot be disabled. 
 
+The supported feature gates apply for all Strimzi operators.
+While a particular feature gate might be used for example only in one of them, it can be configured in all operators.
```

Comment:
"it can be configured in all operators" and maybe we could clarify that will be just ignored where it's not used.